File: branches/darc/Libs/ConfigurationLib/src/configlib/Serializable.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */

package configlib;

import java.io.IOException;

/**
 *
 * @author angr
 */
public interface Serializable
{
    void serialize(ArchiverInOut archive) throws IOException;

}

File: C#/Ops/Archiver/Serializable.cs
///////////////////////////////////////////////////////////
//  Serializable.cs
//  Implementation of the Interface Serializable
//  Created on:      12-nov-2011 21:02:40
//  Author:
///////////////////////////////////////////////////////////

namespace Ops
{
	public interface ISerializable 
    {
		void Serialize(IArchiverInOut archive);
	}

}